charts/renga/charts/jupyterhub/templates/hub/deployment.yaml

```diff
@@ -164,7 +164,7 @@ spec:
           value: {{ $value | quote }}
         {{ end -}}
         - name: GITLAB_HOST
-          value: {{ template "http" . }}://{{ .Values.global.renga.domain }}/gitlab
+          value: {{ template "http" . }}://gitlab.{{ .Values.global.renga.domain }}
```

Comment:
this should be specified as an `extraEnv` and not built in to the chart (#216 implements this change)